src/IO/ReadWriteBufferFromHTTP.h

```diff
@@ -117,14 +148,23 @@ namespace detail
                 request.setChunkedTransferEncoding(true);
 
             for (auto & http_header_entry: http_header_entries)
-            {
                 request.set(std::get<0>(http_header_entry), std::get<1>(http_header_entry));
+
+            if (withPartialContent())
+            {
+                String range_header_value;
+                if (read_range.end)
+                    range_header_value = fmt::format("bytes={}-{}", read_range.begin + bytes_read, *read_range.end);
+                else
+                    range_header_value = fmt::format("bytes={}-", read_range.begin + bytes_read);
+                LOG_TRACE(log, "Adding header: Range: {}", range_header_value);
```

Comment:
```suggestion
                LOG_TEST(log, "Adding header: Range: {}", range_header_value);
```
(I forgot to replace 'trace' with 'test' before committing the test)